### PR TITLE
Change `nuget push` to `dotnet nuget push`

### PR DESCRIPTION
**Pull request by hansmbakker**
This uses [`dotnet nuget push`](https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-nuget-push) rather than `nuget push`.

@matmork can you please verify that the workflow has access to the secret?

### CODE REVIEW

File: .github/workflows/nuget-push.yml [hunk 1]
```diff
@@ -44,11 +44,11 @@ jobs:
...
+      - name: Build packages
         run: |
           dotnet pack -o staging -c Release src/Rive.Android/Rive.Android.csproj
           dotnet pack -o staging -c Release src/Rive.iOS/Rive.iOS.csproj
           dotnet pack -o staging -c Release src/Rive.Maui/Rive.Maui.csproj
 
-      - name: Publish
-        run: nuget push staging/*.nupkg -Source https://api.nuget.org/v3/index.json -ApiKey ${{secrets.NUGET_API_KEY}} -SkipDuplicate -NonInteractive
+      - name: Publish to NuGet
+        run: dotnet nuget push staging/*.nupkg --source https://api.nuget.org/v3/index.json --api-key ${{ secrets.NUGET_API_KEY }} --skip-duplicate
```
Comment (matmork) [inline .github/workflows/nuget-push.yml:54]:
```suggestion
        run: dotnet nuget push "staging/*.nupkg" --source https://api.nuget.org/v3/index.json --api-key ${{ secrets.NUGET_API_KEY }} --skip-duplicate
```

Maybe try add double quotes as https://stackoverflow.com/a/65507712

**Comment by matmork** [comment]
Secret should be added. That doesn't look like the problem

```
Run nuget push staging/*.nupkg -Source https://api.nuget.org/v3/index.json -ApiKey *** -SkipDuplicate -NonInteractive
push: invalid arguments.
usage: NuGet push <package path> [API key] [options]
```

**Review by matmork — APPROVED** [review]

File: .github/workflows/nuget-push.yml (hunk 1 above)
Comment (hansmbakker) [reply .github/workflows/nuget-push.yml:54]: Good point!
image
See https://github.com/NuGet/Home/issues/4393#issuecomment-667618120

**Comment by hansmbakker** [comment]
Likely the issue was `enclosing quotes are required for shells such as bash that perform file globbing` - see https://github.com/NuGet/Home/issues/4393#issuecomment-667618120

image

Source: https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-nuget-push#:~:text=The%20enclosing%20quotes%20are%20required%20for%20shells%20such%20as%20bash%20that%20perform%20file%20globbing.%20For%20more%20information%2C%20see%20NuGet/Home%234393.

**Pull request merged**